### PR TITLE
Add "Edit in codepen" button for component examples.

### PR DESCRIPTION
**Pull request by jonearley**
## What's new?

- Add "Edit in CodePen" option to code examples.
- Add December 2020 updates.

### CODE REVIEW

File: packages/docs/_data/site.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "cloudinary_url": "https://res.cloudinary.com/umich-lib"
+  "cloudinary_url": "https://res.cloudinary.com/umich-lib",
+  "codepen_html_comment": "<!--\nStylesheet and JavaScript included as external assets in CodePen settings.\n\nDocs: https://design-system.lib.umich.edu/\n-->\n\n"
```
Comment (jonearley) [inline packages/docs/_data/site.json:3]: A comment that's used as a reminder in CodePen about configured assets.

File: packages/docs/_includes/layout.njk [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: Documentation for the U-M Library Design System.
   <head>
     <meta charset="utf-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <title>{{ title or eleventyNavigation.key }} | Design System | University of Michigan Library</title>
+    <title>{{ title or eleventyNavigation.key }} — Design System — University of Michigan Library</title>
```
Comment (jonearley) [inline packages/docs/_includes/layout.njk:10]: I can never make up my mind

File: packages/docs/component-pages.njk [hunk 3]
```diff
@@ -31,7 +31,24 @@ eleventyComputed:
...
+      "title": "{{ component.name }} — Design System — University of Michigan Library",
+      "html": "{{ site.codepen_html_comment | safe | replace('"', '\\"') | replace('\'', '&#39;') | replace('\n', '\\n')}} {{ component.example | safe | replace('"', '\\"') | replace('\'', '&#39;') | replace('\n', '\\n') }}",
+      "css_external": "https://unpkg.com/@umich-lib/css@v1/dist/umich-lib.css",
+      "js_external": "https://unpkg.com/@umich-lib/components@v1/dist/umich-lib/umich-lib.js",
+      "layout": "left",
+      "editors": "100"
+    }'
+/>
+    <input type="submit" value="Edit on CodePen" 
+    class="bg-blue-100 text-teal-400 px-4 py-3 mt-3 rounded cursor-pointer font-bold hover:shadow-outline text-base"/>
```
Comment (jonearley) [inline packages/docs/component-pages.njk:49]: Using tailwind to make a button!

File: packages/docs/index.md [hunk 4]
```diff
@@ -83,4 +83,4 @@ Once you’ve added each package shown above, the markup for your HTML page shou
 </html>
 ```
 
-[Edit this example in CodeSandbox](https://codesandbox.io/s/umich-libcomponents-and-umich-libcss-with-unpkg-468rj?fontsize=14&hidenavigation=1&theme=dark)
+<a href="https://codepen.io/team/ulibdd/pen/xxELybM" class="bg-blue-100 text-teal-400 px-4 py-3 mt-3 rounded cursor-pointer font-bold hover:shadow-outline text-base no-underline">Edit on CodePen</a>
```
Comment (jonearley) [inline packages/docs/index.md:86]: Just point to an account CodePen rather than codesandbox.

File: packages/docs/styles/_maybe-tailwind.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.hover\:shadow-outline:hover {
```
Comment (jonearley) [inline packages/docs/styles/_maybe-tailwind.scss:1]: A place to group class names that might be useful to have as a tailwind option some day.

File: packages/docs/updates.md [hunk 6]
```diff
@@ -9,6 +9,11 @@ description: Read the latest updates from the Design System team. We will highli
 
 {{ description }}
 
+## December 2020
```
Comment (jonearley) [inline packages/docs/updates.md:12]: Did anything else happen worth mentioning?

File: packages/docs/styles/_maybe-tailwind.scss (hunk 5 above)
Comment (respinos) [reply packages/docs/styles/_maybe-tailwind.scss:1]: Can you use the tailwind `apply` business here, too?

File: packages/docs/index.md (hunk 4 above)
Comment (respinos) [reply packages/docs/index.md]: This "edit in codepen" link functions differently than the other "edit in codepen" buttons. Is that going to be an issue?

**Review by respinos — APPROVED** [review]
I don't think any of my comments are blockers.

File: packages/docs/updates.md (hunk 6 above)
Comment (bridgetburke) [reply packages/docs/updates.md:12]: I think this is a fine high-level update.

**Review by bridgetburke — APPROVED** [review]

File: packages/docs/styles/_maybe-tailwind.scss (hunk 5 above)
Comment (jonearley) [reply packages/docs/styles/_maybe-tailwind.scss:1]: If we configured docs to process the styles, yeah, but for now just using the CDN. I think this shadow value needs a token too 🤔

File: packages/docs/index.md (hunk 4 above)
Comment (jonearley) [reply packages/docs/index.md]: Good catch, but I'm not too worried about this case.

**Pull request merged**